### DISCUSSION TITLE
Strong Swan File

### DISCUSSION THREAD

**Discussion by Jim97M** [strongswan/strongswan#2077] [General]
Is there a file called secrets.conf in strongswan

**Comment by strongX509** [comment]
With the legacy **`ipsec`** command there is an **`ipsec.secrets`** file whereas with the current **`swanctl`** command there is a a **`secrets`** section in the **`swanctl.conf`** configuration file

**Comment by Jim97M** [comment]
Thank you. Somebody insisted I provide secrets.conf file that reference both our Peer IPs and the PSK
